lang/node-javascript-obfuscator/Makefile

```diff
@@ -0,0 +1,85 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NPM_NAME:=javascript-obfuscator
+PKG_NAME:=node-$(PKG_NPM_NAME)
+PKG_VERSION:=0.18.1
+PKG_RELEASE:=3
+
+PKG_SOURCE:=$(PKG_NPM_NAME)-$(PKG_VERSION).tgz
+PKG_SOURCE_URL:=http://registry.npmjs.org/$(PKG_NPM_NAME)/-/
+PKG_HASH:=3f53199ed892e3faf4ab5726a70181b970b1d1b239f4e6a13d6272cbee02d7cb
+PKG_BUILD_DEPENDS:=node/host
+HOST_BUILD_DEPENDS:=node/host
+PKG_USE_MIPS16:=0
+
+PKG_MAINTAINER:=Zbynek Kocur <zbynek.kocur@fel.cvut.cz>
+PKG_LICENSE:=BSD-2-Clause
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/host-build.mk
```

Comment:
> Do you need a host build?

Yes, I need it. I need to obfuscate my code before it's installing to a target device. 
Is there any other way to do it without using the host build?

Reply:
If that's what you want, you can run it with node.js installed normally on the host OS.

Reply:
Yes, It's true but for mass production and quick deployment of the compilation environment it is more practical to have all dependencies in one package.

Reply:
OK.

This is better to change since "Host/Install" is currently disabled.

Please change it referring to this
 https://github.com/nxhack/openwrt-node-packages/blob/master/node-cpx/Makefile

Reply:
> OK.
> 
> This is better to change since "Host/Install" is currently disabled.
> 
> Please change it referring to this
> https://github.com/nxhack/openwrt-node-packages/blob/master/node-cpx/Makefile

I made changes to your design. Is it correct?

Reply:
LGTM